test/acceptance/go.mod

```diff
@@ -4,8 +4,8 @@ go 1.14
 
 require (
 	github.com/gruntwork-io/terratest v0.31.2
-	github.com/hashicorp/consul/api v1.4.1-0.20201015173526-812fe06d6c64
-	github.com/hashicorp/consul/sdk v0.6.0
+	github.com/hashicorp/consul/api v1.4.1-0.20210415000851-62fcf1ff17cd
+	github.com/hashicorp/consul/sdk v0.7.0
```

Comment:
not sure why this change is needed.

Reply:
We don't strictly need it for the acceptance tests here, but this ensure the acceptance tests run against a version of Consul that does explicitly support the `TransparentProxy` field, even though we do not set it or should allow it to be set. It was done more from a correctness perspective.

Reply:
got it, thanks for clarifying!